libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/marker/LongPressMapComponent.kt

```diff
@@ -8,12 +8,17 @@ import com.mapbox.navigation.dropin.DropInNavigationViewContext
 import com.mapbox.navigation.dropin.component.destination.DestinationAction
 import com.mapbox.navigation.dropin.lifecycle.UIComponent
 import com.mapbox.navigation.dropin.model.Destination
+import com.mapbox.navigation.dropin.util.HapticFeedback
 
 internal class LongPressMapComponent(
     private val mapView: MapView,
-    private val context: DropInNavigationViewContext
+    private val navContext: DropInNavigationViewContext
 ) : UIComponent() {
 
+    private val hapticFeedback by lazy {
+        HapticFeedback.create(navContext.context)
```

Comment:
We can get the context from `mapboxNavigation.navigationOptions.applicationContext`. Let's not use it from `DropInNavigationViewContext` because this has been removed in this [PR](https://github.com/mapbox/mapbox-navigation-android/pull/5594)

Reply:
Updated